### PR TITLE
Add app-root / fe-root to GitHub Actions workflows and kubernetes staging deployment

### PR DESCRIPTION
**Pull request by mcbouslog**
## Package
- app-root

## Linked Issue and/or Talk Post
Continuation from #6010, I think complimentary to zooniverse/static/pull/343.

## Describe your changes
- add app-root to ci-build workflow
- add app-root to code coverage workflow
- add app-root to kubernetes staging deployment

## How to Review
_Helpful explanations that will make your reviewer happy:_
- What user actions should my reviewer step through to review this PR?
- Are there plans for follow up PR’s to further fix this bug or develop this feature?

# Checklist
_PR Creator - Please cater the checklist to fit the review needed for your code changes._
_PR Reviewer - Use the checklist during your review. Each point should be checkmarked or discussed before PR approval._

## General
- [ ] Tests are passing locally and on Github
- [ ] Documentation is up to date and changelog has been updated if appropriate
- [ ] You can `yarn panic && yarn bootstrap` or `docker-compose up --build` and FEM works as expected
- [ ] FEM works in all major desktop browsers: Firefox, Chrome, Edge, Safari (Use Browserstack account as needed)
- [ ] FEM works in a mobile browser

### CODE REVIEW

**Comment by goplayoutside3** [comment]
This looks good to me, but I'll let @zwolf hit the approve button. @mcbouslog do you want to include this in FEM deploy this week?

File: .github/workflows/ci-build.yml
Comment (zwolf) [inline .github/workflows/ci-build.yml:83]: don't need the fe-root build here?

**Review by zwolf — APPROVED** [review]
Look good. Ping me when this is going to merge and I'll watch the staging deploy to ensure that everything comes up correctly. The static PR should be deployed first, and we can confirm everything is routing correctly.

**Comment by mcbouslog** [comment]
I think I'd prefer to merge this after this week's FEM deploy (I think planned for tomorrow, Thursday mid-day). Tentative plan is as follows, subject to suggestion:
1. make small suggestions in this PR
2. merge and deploy related static PR, likely Thursday AM
3. merge which will trigger or manually run staging deploy for this PR, likely Thursday PM

File: .github/workflows/ci-build.yml
Comment (mcbouslog) [reply .github/workflows/ci-build.yml:83]: not yet, no storybooks in fe-root. so far we're just using stories in libraries that we're importing into fe-root (i.e. lib-user, shortly lib-content-pages). we might eventually have fe-root stories, then will have to add here, but not totally sure we will.

**Pull request merged**
